spec/system/summative_assessment_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Assessment' do
+  include_context 'with progress'
+
+  let(:pass_test) do
+    visit training_module_summative_assessment_path('alpha', '1-3-2-1')
+    2.times do
+      choose '5'
+      click_on 'Save and continue'
+    end
+    choose '5'
+    click_on 'Finish test'
+  end
+
+  let(:fail_test) do
+    visit training_module_summative_assessment_path('alpha', '1-3-2-1')
+    2.times do
+      choose '4'
+      click_on 'Save and continue'
+    end
+    choose '4'
+    click_on 'Finish test'
+  end
+
+  before do
+    view_pages_before_summative_assessment(alpha)
+  end
+
+  include_context 'with user'
+
+  context 'when a user has visited each module up to and including a summative assessment' do
+    it 'can click to resume training to visit the summative assessment page' do
+      visit training_module_path('alpha')
+      click_on 'Resume training'
+      expect(page).to have_current_path training_module_summative_assessment_path('alpha', '1-3-2-1'), ignore_query: true
+    end
+  end
+
+  context 'when a user has passed the summative assessment' do
+    it 'reflect on learning link becomes clickable' do
+      pass_test
```

Comment:
rather than it be a `let` the `pass_test` would be better as a `before` block within this context.

Reply:
I can never remember what's allowed in which context. I thought i'd tried this. Clearly not. I'll make the change now